hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsConfigKeys.java

```diff
@@ -184,6 +184,16 @@ public final class HddsConfigKeys {
   // Default Certificate duration to one year.
   public static final String HDDS_X509_DEFAULT_DURATION_DEFAULT = "P365D";
 
+  /**
+   * Duration in days within which before the certificate expiration date the
```

Comment:
This description is hard to understand (at least to me), I'd prefer something like this: "Duration of the grace period in days in which a certificate should be removed before the current one expires."

Reply:
Removed -> renewed,  Agree with the remaining part.

Reply:
Done, the new documentation says:
  /**
   * Duration of the grace period within which a certificate should be
   * renewed before the current one expires.
   * Default is 28 days.
   */